meta-lmp-base/recipes-core/packagegroups/packagegroup-base.bbappend

```diff
@@ -0,0 +1,9 @@
+# Enable using alsa as a DISTRO feature.
+
+PACKAGES += " \
+    ${@bb.utils.contains("DISTRO_FEATURES", "alsa", "packagegroup-base-alsa", "", d)} \
```

Comment:
I think this can probably be dropped as packagegroup-base-alsa is already installed when alsa is available distro features.

We depend on packagegroup-base-extended, which depends on packagegroup-base, which depends on packagegroup-base-alse when it is also available in machine features (when really needed).

Reply:
I still can't find that part in the OE code which "converts" DISTRO features into groups dependencies, but yeah, it works, so I dropped it. Done.